.changes/unreleased/tinymce-TINY-12384-2025-09-16.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: Filtering out Dragster elements in serializer ( elements with classes starting with `ephox-dragster-` )
```

Comment:
No customer would know what dragster is. So maybe it should be more vague just mentioning that some UI elements for table resizing was not properly filtered out.